src/data/swot_db.py

```diff
@@ -1,15 +1,17 @@
 ''' File to define SWOT MongoDB operations. '''
 import pymongo
 
-from src.data.data import _db
+from src.data.data import DatabaseConnection
 from src.data.associates_db import update_associate_swot
 from src.models.swot import SWOT
 
 from src.logging.logger import get_logger
 
 _log = get_logger(__name__)
 
-_swot = _db['swot']
+DB = DatabaseConnection()
```

Comment:
Change this to a one liner so that you cannot export DB. Make it so that DB comes from a single location, even if it is always the same instance.